README.md

```diff
@@ -7,7 +7,7 @@ Webmozart Assert
 [![Total Downloads](https://poser.pugx.org/webmozart/assert/downloads.svg)](https://packagist.org/packages/webmozart/assert)
 [![Dependency Status](https://www.versioneye.com/php/webmozart:assert/1.2.0/badge.svg)](https://www.versioneye.com/php/webmozart:assert/1.2.0)
```

Comment:
You should probably update this line too, no? (cf. commit 4a8bf11547e139e77b651365113fc12850c43d9a)